addons/mail/static/src/web/activity/activity_cell.xml

```diff
@@ -3,9 +3,9 @@
 
     <t t-name="mail.ActivityCell" owl="1">
         <div class="h-100 cursor-pointer p-3" t-on-click="onClick">
-            <div class="h-100 d-flex align-items-center justify-content-center position-relative" t-ref="content">
+            <div class="h-100 d-flex align-items-center justify-content-center gap-1 text-nowrap" t-ref="content">
```

Comment:
For the spacing: We can maybe remove the `gap-1` class and keep the margin class (`ms-2`) you added on the badge? I would choose a side :crossed_swords: and avoid mixing `gap` and margins.

As I don't know if there should be the same spacing between each child of that element, better use margins.

You can green the task once the code is updated :smiley: